test/e2e/serial/tests/configuration.go

```diff
@@ -71,7 +71,7 @@ var _ = Describe("[serial][disruptive][slow] numaresources configuration managem
 		Expect(serialconfig.Config.Ready()).To(BeTrue(), "NUMA fixture initialization failed")
 
 		var err error
-		fxt, err = e2efixture.Setup("e2e-test-configuration")
+		fxt, err = e2efixture.Setup("e2e-test-configuration", serialconfig.Config.NRTList)
```

Comment:
I wonder if we can just pass `serialconfig.Config`. But this generalization should probably wait for a future PR

Reply:
we certainly can, and I think since we are already on it let's have it at once, unless you see a strong reason not to. I'll have it updated to `serialconfig.Config` let me know if it works for you.

Reply:
certainly we can, but this will require a change in the structure of the suite. the minimal change would be moving E2EConfig struct to somewhere else than under serial/ (for example utils/configuration/configuration.go or some other file with known types across the project - not necessarily e2e). on one hand this enables us passing the whole suite config to anywhere, but OTOH its only instance would be initialized in serial. I don't see it suites anywhere else other than serial though. 

we cannot simply pass the serialconfig.Config to utils/fixture because this will create an import loop. and if otherwise we could have just used serialconfig.Config.NRTList directly in utils/fixture.

Reply:
right, there could be an import loop. I think there's no need to embark in a large refactoring now and we can proceed with the direction you already took.